arlo-client/src/components/AuditFlow/index.test.tsx

```diff
@@ -49,15 +55,30 @@ describe('AuditFlow ballot interaction', () => {
   })
 
   it('renders board table with no ballots', async () => {
-    const { queryByText, container } = render(
-      <StaticRouter {...staticRouteProps}>
-        <AuditFlow {...routeProps} />
-      </StaticRouter>
+    apiMock.mockImplementation(
+      async (endpoint: string): Promise<IAudit | { ballots: IBallot[] }> => {
+        switch (endpoint) {
+          case '/election/1/audit/status':
+            memberDummy.jurisdictions[0].auditBoards = [dummyBoard[1]]
```

Comment:
This test now has side effects that could affect other tests. Would something like this be better?

```ts
return {
  ...memberDummy,
  jurisdictions: [
    {
      ...memberDummy.jurisdictions[0],
      auditBoards: [dummyBoard[1]]
    },
    ...memberDummy.jurisdictions.slice(1)
  ]
}
```